### ISSUE TITLE
Hello, thank you for your sharing. But my running is not normal. I don't know the reason.

### ISSUE THREAD

**Issue by tianqingqian** [xfmao/ATDA#1]
Iter 1 of Epoch 2  labeling_cost 0.103619 targetspecific_cost 0.035066
Iter 1 of Epoch 2  labeling_cost 0.561946 targetspecific_cost nan

**Comment by tianqingqian** [comment]
I slove this problem by modify the loss ：
self.F1_loss = -tf.reduce_mean(self.y_ * tf.log(self.F1_out+1e-10))
self.F2_loss = -tf.reduce_mean(self.y_*tf.log(self.F2_out+1e-10))
self.Ft_loss = -tf.reduce_mean(self.y_*tf.log(self.Ft_out+1e-10))
but my acc is about 43%, could you give us  your model and weight? Thank you very much.
